src/main/kotlin/nl/avisi/structurizr/site/generatr/site/SiteGenerator.kt

```diff
@@ -74,11 +74,12 @@ private fun copyAssets(assetsDir: File, exportDir: File) {
 }
 
 private fun generateStyle(context: GeneratorContext, exportDir: File) {
+    val branchDir = File(exportDir, context.currentBranch)
```

Comment:
Let’s move this to line 63 as

```kotlin
val branchDir = File(exportDir, currentBranch)
```

then we can just pass branchDir directly into `deleteOldHashes`, `copyAssets`, `generateStyle` and `generateHtmlFiles`.

`deleteHashes` has the same issue that it is applied for all branches, but since the hashes are only used for hot reloading with serve and a single branch, it never surfaced.

Reply:
I wonder if we even need `GeneratorContext.currentBranch` after this change. May be it is unused now and can be removed.